### PR TITLE
[CI] Refresh GitHub workflows

### LINKED ISSUE #18: [CI] Refresh GitHub workflows
### Current Behavior
This repo is created quite some time ago and contains our antiquated content. 

### Implementation
Update each [Meshery Adapter for Nighthawk workflows](https://github.com/meshery/meshery-nighthawk/tree/master/.github) as appropriate. 
Please refresh from [Meshery Adapter for Istio workflows](https://github.com/meshery/meshery-istio/tree/master/.github) as appropriate.

#### Reference:

- [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)

### PR DESCRIPTION
**Pull request by gyohuangxin**
**Description**

This PR fixes #18 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 1]
```diff
@@ -2,18 +2,21 @@
...
 
+
 ---
 
-#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
-- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
-- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
+#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
+- 📝 [Meshery Adapters Spreadsheet](https://docs.google.com/spreadsheets/d/1rGGpSXC68iDJzNRU-qZIRQsZUwkt1qLFg7JCs4pfvbU/edit?usp=sharing)
+- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
```
Comment (hamza-mohd) [inline .github/ISSUE_TEMPLATE/documentation.md:18]: Awesome

File: .github/config.yml [hunk 2]
```diff
@@ -15,17 +15,16 @@ newPRWelcomeComment: >
...
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-  Thanks for your contribution to Meshery! :tada:
+  Thanks for your contribution to the Meshery! :tada:
    
   <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery-perf/master/.github/welcome/Meshery-celebration-Dark.svg">
-  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery-perf/master/.github/welcome/Meshery-celebration-Light.svg">
-  <img alt="Meshery Logo" src="">
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery-nighthawk/master/.github/welcome/meshery-darkmode-celebration.svg">
```
Comment (leecalcote) [inline .github/config.yml:21]: I went ahead and opened https://github.com/meshery/meshery-nighthawk/issues/22 to bring in a bit more Nighthawk to the repo.

File: .github/release-drafter.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-name-template: 'Meshery Perf v$NEXT_PATCH_VERSION'
+name-template: 'Meshery Adapter for Nighthawk v$NEXT_PATCH_VERSION'
```
Comment (leecalcote) [inline .github/release-drafter.yml:1]: Wow. You nailed this. Perfect.

**Review by leecalcote — APPROVED** [review]
I think we're good to go. If anything was missed, it won't break anything that isn't broken already. 😁

**Pull request merged**